src/System.Runtime.Serialization.Primitives/src/System.Runtime.Serialization.Primitives.csproj

```diff
@@ -22,7 +23,15 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcore50aot_Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcore50aot_Release|AnyCPU'" />
   <ItemGroup  Condition="'$(TargetGroup)'!='net46'">
-    <Compile Include=".\**\*.cs" />
+    <Compile Include=".\**\*.cs"
+             Exclude=".\System\Runtime\Serialization\FormatterServices.cs;
+                      .\System\Runtime\Serialization\StreamingContext.cs"
+       />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'=='netcore50'">
+    <!-- throws NotImplementedException -->
+    <Compile Include=".\System\Runtime\Serialization\FormatterServices.cs" />
+    <Compile Include=".\System\Runtime\Serialization\StreamingContext.cs" />
```

Comment:
netcore50 is generally the .NET Native implementation, not the CoreCLR implementation. In this case, `netcore50` is not a partial facade, so everything in the contract needs to be implemented here.